### PR TITLE
Catch exception if remove_veth throws one

### PR DESCRIPTION
**Pull request by TrimBiggs**

### CODE REVIEW

File: calico_containers/calico_ctl/container.py [hunk 1]
```diff
@@ -235,8 +235,12 @@ def container_remove(container_name):
                 # enforcing assignments strictly in datastore.py.
                 client.unassign_address(pool, ip)
 
-    # Remove the endpoint
-    netns.remove_veth(endpoint.name)
+    try:
+        # Remove the interface if it exists
+        netns.remove_veth(endpoint.name)
+    except CalledProcessError:
+        print "Could not remove calico interface"
```
Comment (robbrockbank) [inline calico_containers/calico_ctl/container.py]: Capital C.

Put in the interface name otherwise there is not much to go on to help debug.

**Comment by robbrockbank** [comment]
What's the plan for picking up the change to libcalico?

**Comment by TrimBiggs** [comment]
Once the libcalico change goes in, I'll set a tag on libcalico and update install.sh/requirements.txt where libcalico is installed to use the new tag.

**Comment by TrimBiggs** [comment]
This tag will be included as a new release.

File: calico_containers/calico_ctl/container.py [hunk 2]
```diff
@@ -235,8 +234,12 @@ def container_remove(container_name):
                 # enforcing assignments strictly in datastore.py.
                 client.unassign_address(pool, ip)
 
-    # Remove the endpoint
-    netns.remove_veth(endpoint.name)
+    try:
+        # Remove the interface if it exists
+        netns.remove_veth(endpoint.name)
+    except CalledProcessError:
+        print "Could not remove Calico interface"
```
Comment (robbrockbank) [inline calico_containers/calico_ctl/container.py]: I would put the interface name in here, otherwise it doesn't give much to help debug.

**Comment by robbrockbank** [comment]
One comment, then LGTM

**Pull request merged**